src/sentry/templates/sentry/emails/relocate_account.txt

```diff
@@ -0,0 +1,8 @@
+The following Sentry organizations that you are a member of have been migrated onto sentry.io:
```

Comment:
Honestly, I'm not sure why we need the `.txt` file, the tests were complaining when I didn't add it though. Maybe it could be good to rope someone in that's familiar with this flow for a review

Reply:
Not all Email clients read HTML. We also use the text content in tests.